OpenRA.sln

```diff
@@ -107,7 +109,7 @@ Global
 		$0.DotNetNamingPolicy = $1
 		$1.DirectoryNamespaceAssociation = None
 		$1.ResourceNamePolicy = FileFormatDefault
-		$0.TextStylePolicy = $2
+		$0.TextStylePolicy = $7

```

Comment:
These changes may be IDEs fighting each other. It is usually best to avoid them. Less noise in the git history (and fewer rebase problems). Try to only stage the line `mods\cnc\maps\nod02a\nod02a.lua = mods\cnc\maps\nod02a\nod02a.lua` and revert the rest after committing.